### PR TITLE
SASS-682 completed dependancies

### PR DESCRIPTION
**Pull request by lz1kwn**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [ ]  Have you checked the PR Builder passes?
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-import sbt.Keys.libraryDependencies
 import sbt._
 
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-frontend-play-27" % "3.4.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.49.0-play-27"
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.3.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.68.0-play-28",
```
Comment (matt712) [inline project/AppDependencies.scala:8]: This could be  0.69.0-play-28

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
-resolvers += "HMRC Releases" at "http://dl.bintray.com/hmrc/releases"
+resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.14.0")
```
Comment (matt712) [inline project/plugins.sbt:6]: Sbt AutoBuild can be bumped up to 3.0.0,
Also, we should be using these resolvers now 
resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)

**Review by matt712 — CHANGES_REQUESTED** [review]

**Review by matt712 — APPROVED** [review]

**Comment by matt712** [comment]
Somethings in pipeline right now

**Comment by matt712** [comment]
I missed a couple of dependency bumps;
 “uk.gov.hmrc”             %% “govuk-template”             % “5.66.0-play-27" can be 5.68.0
"org.scoverage" % "sbt-scoverage" % "1.6.1"   can be 1.8.2
com.typesafe.play” % “sbt-plugin” % “2.7.5" should be 2.8.8
oh and sassify can be “org.irundaia.sbt” % “sbt-sassify” % “1.5.1”

**Review by BenCosfordHMRC — APPROVED** [review]

**Pull request merged**